helix-view/src/editor.rs

```diff
@@ -1079,6 +1079,12 @@ impl Editor {
         self._refresh();
     }
 
+    pub fn open_url(&mut self, url: lsp::Url) {
```

Comment:
This could probably just all go into the command. I'm not sure I understand why this is an `Editor` function.